imports/react-ui/settings/knowledgebase/components/topic/topic.js

```diff
@@ -0,0 +1,192 @@
+import React, { PropTypes } from 'react';
+import Select from 'react-select-plus';
+import {
+  FormGroup,
+  ControlLabel,
+  FormControl,
+  Button,
+  ButtonToolbar,
+  Modal,
+} from 'react-bootstrap';
+import ReactMarkdown from 'react-markdown';
+import SelectBrand from '../SelectBrand';
+import CopyToClipboard from 'react-copy-to-clipboard';
+import { CommonItem } from '../common';
+
+class KbTopic extends CommonItem {
+  constructor(props, context) {
+    super(props, context);
+
+    let code = '';
+
+    // showed install code automatically in edit mode
+    if (props.item) {
+      code = this.constructor.getInstallCode(props.item._id);
```

Comment:
Can't be just `this.getInstallCode()`?

Reply:
cant get properties inside a static method